model/flow/identity.go

```diff
@@ -274,6 +274,18 @@ func (il IdentityList) Order(less IdentityOrder) IdentityList {
 	return dup
 }
 
+// Ordered returns whether the list is ordered by the input ordering.
+func (il IdentityList) Ordered(less IdentityOrder) bool {
```

Comment:
Is it assumed that all elements of the receiver `il` are valid elements of the ordering `less`? I mean is the call `less(a,b)` always safe?

Reply:
It is safe. Orderings should be at least consistent when applied to any identity list (they are just a function `(a, b Identity) -> Bool`)